app/api/models/errors/CommonMtdErrors.scala

```diff
@@ -120,7 +120,7 @@ object ClientNotAuthorisedError    extends MtdError("CLIENT_OR_AGENT_NOT_AUTHORI
 
 //Stub errors
 object RuleIncorrectGovTestScenarioError
-    extends MtdError(code = "RULE_INCORRECT_GOV_TEST_SCENARIO", message = "The Gov-Test-Scenario was not found", NOT_FOUND)
+    extends MtdError(code = "RULE_INCORRECT_GOV_TEST_SCENARIO", message = "The Gov-Test-Scenario was not found", BAD_REQUEST)
 
```

Comment:
RuleIncorrectGovTestScenarioError should have BAD_REQUEST status code; this was overlooked previously.